subgraph.template.yaml

```diff
@@ -1243,6 +1243,31 @@ dataSources:
           handler: handleSetBooleanValueProjectConfig
       file: ./src/minter-suite-mapping.ts
   {{/minterMerkleV1Contracts}}
+  {{#adminACLV0Contracts}}
+  - kind: ethereum/contract
+    name: 'AdminACLV0-{{address}}'
+    network: {{network}}
+    source:
+      address: '{{address}}'
+      abi: AdminACLV0
+      startBlock: {{startBlock}}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.5
+      language: wasm/assemblyscript
+      entities:
+        - Project
+        - Contract
+      abis:
```

Comment:
I think you'll need to include the GenArt721CoreV3 in the ABIs list here as well since you make GenArt721CoreV3 contract calls when the SuperAdminTransferred event handler gets called. Don't think this breaks the tests but will break when actually running.

Reply:
SGTM! updated and pushed! I also added the IAdminACLV0 to the core contract portion of the template, but maybe we can chat more about exactly why these are needed (I don't have a great mental model of what these ABIs do...)